modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/edit_results_rankings.jsp

```diff
@@ -64,6 +64,7 @@ String companyId = ParamUtil.getString(request, "companyId");
 			cancelUrl: '<%= HtmlUtil.escape(redirect) %>',
 			fetchDocumentsHiddenUrl: '<%= hiddenResultsRankingResourceURL %>',
 			fetchDocumentsUrl: '<%= resultsRankingResourceURL %>',
+			initialAliases: '<%= HtmlUtil.escape(aliases) %>',
```

Comment:
`aliases` isn't defined so this will cause errors. We can use an empty array for now until the backend is updated to support retrieving aliases.